website/integrations/services/jenkins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Jenkins
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Jenkins
+
+> The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.
+>
+> -- https://www.jenkins.io/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `jenkins.company` is the FQDN of the Service install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+Create an application in authentik. Create an OAuth2/OpenID provider with the following parameters:
```

Comment:
```suggestion
Create an OAuth2/OpenID provider with the following parameters:
```

Reply:
I'd suggest you first have them create the provider, and then below create the app.